kubernetes/resources/honeydipper-job.yaml.tmpl

```diff
@@ -165,6 +165,9 @@ spec:
       {{- with .ctx.pod_securityContext }}
       securityContext: {{ . | toJson }}
       {{- end }}
+      {{- with .ctx.serviceAccountName }}
```

Comment:
Does this skip evaluation if the property isn't set?

Reply:
right, that is how `with` statement in go template works.

https://pkg.go.dev/text/template#hdr-Actions